pkg/container/qemu_runner.go

```diff
@@ -65,6 +65,10 @@ const (
 	defaultDiskSize = "50Gi"
 )
 
+var sshClient *ssh.Client
+var workspaceClient *ssh.Client
+var qemuPID int
```

Comment:
I think we should make these fields on `qemu`.

My concern is that these potentially carry multiple values over the process lifetime if multiple test pods get run, and if we were smart enough to run them in parallel then they could end up racing and conflict!

Reply:
done